fleet_adapter_mir/fleet_adapter_mir.py

```diff
@@ -196,6 +196,20 @@ def create_robot_command_handles(config, handle_data, dry_run=False):
 
             robot.load_mir_missions()
             robot.load_mir_positions()
+
+            # Check that the MiR fleet has defined the variable move mission,
+            # that this adapter will use repeatedly with varying parameters.
+            variable_move_mission = mir_config['variable_move_mission']
```

Comment:
Just a very minor suggestion: Instead of calling this `variable_move_mission`, I'd suggest calling it `rmf_follow_path_mission`. I think that would express its purpose more clearly.

Reply:
https://github.com/osrf/fleet_adapter_mir/pull/16/commits/81d5cb260948691d49ff2f6af83b6b84a7c4844d, good call, I changed the name to `rmf_move_mission`, since multiple waypoint path commands are not supported afaik.